docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -72,3 +72,17 @@ items:
       kubernetes.io/hostname: ocp-worker-0
       node-role.kubernetes.io/worker: ""
 ```
+
+ifeval::["{build}" != "downstream"]
```

Comment:
I dunno how docs team ultimately works, but I wonder if some TODO flag should be added here so that we don't forget to remove the `ifeval` from here when the rest of IPv6 docs are ready.

Reply:
Done